languages/csharp/exercises/concept/structs/.docs/after.md

```diff
@@ -0,0 +1,89 @@
+C# `struct`s are closely related `class`s. They have state and behavior. They can have the same kinds of members: constructors, methods, fields, properties, etc.
+
+Fields and properties can be simple types, `struct`s and reference types. `struct`s observe the same rules about scope, read/write rules and access levels as do `class`s.
+
+```csharp
+enum Unit
+{
+    Kg,
+    Lb
+}
+struct Weight
+{
+    private double count;
+    private Unit unit;
+
+    public Weight(double count, Unit unit)
+    {
+        this.count = count;
+        this.unit = unit;
+    }
+
+    public override string ToString()
+    {
+        return count.ToString() + unit.ToString();
+    }
+}
+
+new Weight(77.5, Unit.Kg).ToString();
+// => "77.6Kg"
+```
+
+One of the main things to remember is that when one struct is assigned to a variable or passed as a parameter the values are copied across so changes to he original variable will not affect the copied one and vice versa. In summary, `struct`s are **value types**.
+
+This [article][class-or-struct] discusses the differences between `struct`s and `class`s. You will see from the article that `struct`s tend to be lightweight and [immutable][structs-immutable] although this guidance is not enforced (by default) by the compiler or runtime.
```

Comment:
Would it make sense to also note that a struct will always have a default constructor, even if you specify a custom one? This is different from classes.

Reply:
Done